### ISSUE TITLE
-v flag is setting verbosity lower than it is defined in conf-file

### ISSUE THREAD

**Issue by bebehei** [rsnapshot/rsnapshot#47]
If I define the verbosity in the rsnapshot.conf with `verbose 5` and invoke rsnapshot with the -v flag, the verbosity is set down to 3. IMO in this case the -v flag is confusing, as I want to have more chatty output, but in reality it limits it.

The behaviour to achieve should be, to set verbose only if the level is higher than it is right now.

**Comment by myrdd** [comment]
> The behaviour to achieve should be, to set verbose only if the level is higher than it is right now.

I would agree, but there might be use cases in which you explicitely want less verbosity. One solution could be not to change the current behaviour, but to output a message that the log level has been reduced. What do you think?

**Comment by bebehei** [comment]
Thanks for the objection. I've got a twisted opinion about that. I can't judge right now, what the proper solution would be.

The reason for this bug-report is, that I tested PR #46, and set the verbosity in log-file to 5, but I have forgotten, that I had the -v flag set. The method print_msg logged only to the logfile which was confusing. That's all for now. I think we have to fix other problems at first, but I let this issue open.
